collector/src/bin/rustc-perf-collector/execute.rs

```diff
@@ -276,7 +281,11 @@ impl<'a> CargoProcess<'a> {
                     ));
                 }
 
-                profiler.subcommand()
+                if self.build_kind == BuildKind::Doc {
+                    "doc"
```

Comment:
Hm, this seems like the wrong subcommand -- we'll probably want `rustdoc` not `doc` as we want the same behavior of checking just the last crate (and not all deps).

Reply:
Ooh, I didn't know `rustdoc` was a thing! That will let me get rid of the `RUSTFLAGS` hack too :)